web/views/student.py

```diff
@@ -824,42 +818,73 @@ def dump_comment(comment):
         except KeyError as e:
             logging.exception(e)
 
-    # add comments from llm summary
-    if is_teacher(request.user):  # Currently only teachers can view LLM summary comments
-        llm_summary = submit_data.ai_review.summary
-
-        if len(llm_summary) > 0:
-            summary_comments.append(
-                {
-                    "id": -1,
-                    "author": AI_REVIEW_COMMENT_AUTHOR,
-                    "text": llm_summary,
-                    "can_edit": False,
-                    "type": AI_REVIEW_COMMENT_TYPE,
-                    "url": None,
-                }
+    # Append review summary to comments
+    if submit_data.ai_review:
```

Comment:
Shouldn't you only show pending suggestions here? If a suggestion becomes accepted, you also create a comment, so you would return both an accepted suggestion and a comment for the same file/line?

Reply:
Sorry, didn't notice it, nevermind.